wasm/generateInterface.js

```diff
@@ -167,6 +203,10 @@ function generateManifest (interfaceManifest, opts) {
   for (let opcode in interfaceManifest) {
     const op = interfaceManifest[opcode]
       // generate the import params
+    if (opcode == 'CALLCODE') {
+      debugger
```

Comment:
Why is this needed?

Reply:
Leftover artifact from testing on my own machine